DB/dbo/PostStatsView.sql

```diff
@@ -0,0 +1,25 @@
+﻿CREATE VIEW [dbo].[poststatsview] as
+with cte_Users as (select Username from Users)
+,cte_Email as (select username, Email from users)
+,cte_First_post as (select authorusername, min(CreatedAt) First_Post from articles group by AuthorUsername)
+,cte_Last_Post as (select authorusername, max(CreatedAt) Last_Post from articles group by authorusername)
+,cte_Article_Count as (select authorusername, count(id) Article_Count from Articles group by authorusername)
+,cte_Distinct_Articles as (select authorusername, count(body)-count(distinct body) Distinct_Articles from Articles group by authorusername)
+,cte_First_Comment as (select username, min(createdat) First_Comment from comments group by username)
+,cte_Last_Comment as (select username, max(createdat) Last_Comment from Comments group by username)
+,cte_Comment_Count as (select username, count(body) Comment_Count from Comments group by username)
+,cte_Deleted_Comments as (select username, count(DeletedAt) Deleted_Comments from comments group by username)
+,cte_Comments_Own_Post as (select comments.username, count(comments.Username) Comments_Own_Post from Comments 
+join articles on comments.ArticleId=articles.id where comments.username=articles.authorusername group by username)
```

Comment:
Move select to a new line, indented. indent join so it lines up with its select. Put the where and group by both on their own lines with the same indent.